### PR TITLE
Add blog about workshop NL

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Almost same as: http://www.defactolearning.de/blog/workshop-e-learning-zukunftsthemen/

### CODE REVIEW

File: source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Workshop “Hoe je met e-learning je toekomst vorm kunt geven"
-date: 2017-02-21 08:00 CEST
```
Comment (fatboypunk) [inline source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown]: Does this actually matter that much btw?

File: source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Wat is e-learning nu eigenlijk precies?
+
+E-Learning is alles wat digitaal wordt bekeken of afgespeeld met als doel kennis over te dragen, medewerkers te stimuleren in hun ontwikkeling of medewerkers te testen. In de huidige wirwar van mogelijkheden is het niet gemakkelijk je weg te vinden om het beste e-learningmateriaal uit te zoeken.
+
+Hoe bepaal je welk aanbod voor jou het beste is en waar vind je vervolgens de juiste partners om dit materiaal te leveren? Of als je de keuze hebt gemaakt zelf materiaal te ontwikkelen, hoe doe je dit dan? Waar begin je, wie zou het kunnen en op welke termijn?
+
+Al met al vragen die verder gaan dan sec de aanschaf van een e-learning module of het uitkiezen van een aanbieder van e-learning materiaal. Vragen die vooral te maken hebben met de algemene visie op het opleidingsbeleid binnen de organisatie. Is die visie niet helder, dan wordt de keuze voor wel of geen e-learning lastig. De vervolgstappen om te kiezen voor een e-learning methode worden zo mogelijk nog lastiger.
```
Comment (fatboypunk) [inline source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown]: `vragen die verder gaan dan sec de aanschaf` don't understand this line

Comment (RonaldDijkstra) [inline source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown]: > verder gaan dan alleen de aanschaf

better?

File: source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown (hunk 1 above)
Comment (RonaldDijkstra) [inline source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown]: No, not really really

File: source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Kennismanagement
+
+In iedere organisatie is veel kennis bij medewerkers aanwezig. Vaak blijft deze kennis onderbelicht en onvoldoende benut. Men weet doorgaans simpelweg niet wat collega’s weten en lang niet altijd is bekend wie de specialist is binnen een bedrijf op het gebied van een bepaald thema.
+
+Bovendien ontwikkelen collega’s zich in hun vrije tijd en dat kan bijdragen aan verbetering van processen of verbreden van kennis binnen organisaties. Het hebben van een Kwaliteitspaspoort voor medewerkers waarin zij kunnen aantonen wat hun kwalificaties en competenties zijn is daarom een zeer handig hulpmiddel.
+
+Het Kwaliteitspaspoort is een centrale plek waarin elke medewerker de mogelijkheid heeft zijn of haar eigen e-portfolio te beheren en te optimaliseren, onafhankelijk van tijd en plaats. De aanschaf van een dergelijk systeem zal uiteraard moeten passen bij de visie op opleiden binnen de organisatie.
```
Comment (fatboypunk) [inline source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown]: do we want to link this?

Comment (RonaldDijkstra) [inline source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown]: Yes we do

File: source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown (hunk 2 above)
Comment (fatboypunk) [inline source/blog/nl/2017-02-22-met-e-learning-toekomst-vormgeven.html.markdown]: yes

**Comment by RonaldDijkstra** [comment]
- added link KPP
- changed 'sec' sentence to 'verder gaan dan alleen de aanschaf' 
- removed the last parts about ZeQ and Defacto because it felt not in place 
- removed the line before closing the post

**Pull request merged**